src/FakeERC20.sol

```diff
@@ -2,6 +2,7 @@
 pragma solidity ^0.8.20;
 
 import {ERC20Votes} from "openzeppelin/token/ERC20/extensions/ERC20Votes.sol";
+
```

Comment:
nit: can undo this change to avoid unneccessary diff on this file

Reply:
https://github.com/ScopeLift/l2-flexible-voting/pull/82/commits/c4ce2317d5e8abeea931f92253011b96e68d0bda